### ISSUE TITLE
fw 4.2.2 exit/crash - no trap files produced

### ISSUE THREAD

**Issue by ecomstation** [bitwiseworks/InJoy#49]
updated injoy firewall to version 4.2.2 i still have crashes in some moments 
firewall is binded to an intel pro 100 S (no multimac) 
PRO/100 Network Connection Driver v4.57 112304
Copyright 1995-2004, Intel Corporation.  All Rights Reserved.

i've no .TRP files 

errors.err in \firewall dir:

Wed Jul 12 11:34:48 2017 - tid#1: Killed by signal! (C0010003)

firewall.log:

2017/07/12 11:34:59  FIREWALL PLUGIN: Shutting down!
2017/07/12 11:40:55  FIREWALL PLUGIN: Initialized successfully
2017/07/12 11:40:55  FIREWALL PLUGIN: Version 4.2, build timestamp May 22 2014 1
3:49:46
2017/07/12 11:40:55  FIREWALL PLUGIN: + Dynamic firewall, + URL handling, + Safe
-Mail, + Traffic Shaper
2017/07/12 11:40:55  FIREWALL PLUGIN: Configuration successfully loaded
2017/07/12 11:40:55  FIREWALL PLUGIN: Ready...

activity.log:

IP Address not configured, using 1.1.1.1 as a temporary address.
FW: new IP address :[1.1.1.1]
FW: new IP address :OK
FW: new IP address :[192.168.10.2]
FW: new IP address :OK
IP Address not configured, using 1.1.1.1 as a temporary address.
FW: new IP address :[1.1.1.1]
FW: new IP address :OK
FW: new IP address :[192.168.10.2]
FW: new IP address :OK

protocol.ini:

[PROT_MAN]

   DRIVERNAME = PROTMAN$

[IBMLXCFG]

   tcpip_nif = tcpip.nif
   FXWRAP_nif = fxwrap.nif
   E100BEO2_nif = E100BEO2.nif
   B57_nif = B57.nif

[tcpip_nif]

   DriverName = TCPIP$
   Bindings   = FXWRAP_nif,B57_nif

[E100BEO2_nif]

   DriverName = E100B$
   SPEED = 0
   FORCEDUPLEX = 0
   TRANSMITBUFFERS = 20
   RECEIVEBUFFERS = 20

[B57_nif]

   DriverName = B57$

[FXWRAP_nif]

   Drivername = FXWRAP1$
   Bindings   = E100BEO2_nif

inetcfg:

#Inetcfg:       CURRENT DEFAULT MINIMUM MAXIMUM

keepalive       300     7800    0       7800    KeepAlive (sec)
tcpswinsize     64240   32768   1024    246723  TCP SendWindow Size
tcprwinsize     64240   32768   1024    246723  TCP RecvWindow Size
udpswinsize     9216    9216    1024    246723  UDP SendWindow Size
udprwinsize     41600   41600   1024    246723  UDP RecvWindow Size
lingertime      120     120     0       65535   Linger Time (sec)
probecnt        8       8       1       8       Number Of KeepAlive Probes
ipforward       1       0       0       1       IP Forwarding flag ON/OFF
mtudiscover     0       1       0       1       TCP Path MTU Discovery ON/OFF
arpkillc        1200    1200    180     15300   ARP Complete TimeOut (sec)
arpkilli        180     180     60      1200    ARP InComplete TimeOut (sec)
lipcstswinsize  10240   10240   1024    246723  LIPC Stream SendWindow Size
lipcstrwinsize  10240   10240   1024    246723  LIPC Stream RecvWindow Size
lipcdgswinsize  2048    2048    1024    246723  LIPC Dgram SendWindow Size
lipcdgrwinsize  4096    4096    1024    246723  LIPC Dgram RecvWindow Size
synattack       0       0       0       1       SYN Attack Flag ON/OFF
syncookie       0       0       0       1       SYN Cookie Flag ON/OFF
firewall        0       0       0       1       FIREWALL ON/OFF
multidefrt      0       1       0       1       Multiple Default Routes ON/OFF
winscale        1       1       0       1       TCP Window Scale ON/OFF
timestmp        1       0       0       1       TCP TimeStamps ON/OFF
realslow        20      20      1       100     TCP TW Q Slow Timeout ticks
perfhttp        1       1       0       1       Fast Path HTTP ON/OFF
tcpttl          64      64      1       255     TCP  TTL (hops)
udpttl          64      64      1       255     UDP  TTL (hops)
icmpttl         64      64      1       255     ICMP TTL (hops)
fragttl         60      60      1       255     Fragment TTL (sec)
reusetw         0       1       0       1       Reuse Time Wait ON/OFF
msl             30      30      1       16384   TCP MaxSegLifeTime (sec)
cc              0       0       0       1       TTCP function ON/OFF
sack            0       0       0       1       Selective ACK ON/OFF

**Comment by ydario** [comment]
To produce a .TRP file the system exceptq is required, you can install it using 'yum install exceptq' or manually installing this zip file http://rpm.netlabs.org/release/00/zip/exceptq-7_11_3-1_oc00.zip. yum method is preferred to allow automatic updates.

**Comment by ecomstation** [comment]
put the exe under path and 2 dll under libpath

do in need also this entry in config.sys?
SET EXCEPTQ=Z

**Comment by ydario** [comment]
No , that is not necessary.

**Comment by ecomstation** [comment]
______________________________________________________________________

 Exception Report - created 2017/07/14 12:23:52
______________________________________________________________________

 InJoy Firewall 4.2.2 GA

 OS2/eCS Version:  2.45
 # of Processors:  2
 Physical Memory:  2558 mb
 Virt Addr Limit:  1536 mb
 Exceptq Version:  7.10 (Mar  1 2011)

______________________________________________________________________

 Exception C0000005 - Access Violation
______________________________________________________________________

 Process:  C:\UTIL\FIREWALL\GATEWAY.EXE
 PID:      6FC (1788)
 TID:      14 (20)
 Priority: 20E

 Filename: C:\UTIL\FIREWALL\IPSEC.DLL
 Address:  005B:1E54573C (0001:0000573C)
 Cause:    Attempted to read from 18E30020
           (unallocated memory)

______________________________________________________________________

 Failing Instruction
______________________________________________________________________

 1E54572A  CALL 0x1e557c60         (e8 31250100)
 1E54572F  TEST EAX, EAX           (85c0)
 1E545731  JNZ  0x1e545938         (0f85 01020000)
 1E545737  MOV  EAX, [0x186c81b4]  (a1 b4816c18)
 1E54573C >MOV  EBX, [EAX+0x20]    (8b58 20)
 1E54573F  TEST EBX, EBX           (85db)
 1E545741  JNZ  0x1e54590c         (0f85 c5010000)
 1E545747  MOV  EDX, [EAX+0x14]    (8b50 14)

______________________________________________________________________

 Registers
______________________________________________________________________

 EAX : 18E30000   EBX  : 026D7E74   ECX : 00000000   EDX  : 00000002
 ESI : 00000001   EDI  : 00000002
 ESP : 026D7C2C   EBP  : 006109D9   EIP : 1E54573C   EFLG : 00010246
 CS  : 005B       CSLIM: FFFFFFFF   SS  : 0053       SSLIM: FFFFFFFF
 DS  : 0053       ES   : 0053       FS  : 150B       GS   : 0000

 EAX : read/write memory at 0002:00000000 in DISTORM
 EBX : read/write memory on this thread's stack
 ECX : not a valid address
 EDX : not a valid address
 ESI : not a valid address
 EDI : not a valid address

______________________________________________________________________

 Stack Info for Thread 14
______________________________________________________________________

   Size       Base        ESP         Max         Top
 00008000   026D8000 -> 026D7C2C -> 026D6000 -> 026D0000

______________________________________________________________________

 Call Stack
______________________________________________________________________

   EBP     Address    Module     Obj:Offset    Nearest Public Symbol
 --------  ---------  --------  -------------  -----------------------
 Trap  ->  1E54573C   IPSEC     0001:0000573C 


______________________________________________________________________

 Labels on the Stack
______________________________________________________________________

   ESP     Address    Module     Obj:Offset    Nearest Public Symbol
 --------  ---------  --------  -------------  -----------------------
 026D7C2C  1E545527   IPSEC     0001:00005527 
 026D7C68  1E555A66   IPSEC     0001:00015A66 
 026D7CC8  00022B11   GATEWAY   0001:00012B11 
 026D7CF8  00022E4D   GATEWAY   0001:00012E4D 
 026D7D0C  1EA02561   FXUTIL    0001:00002561 
 026D7D28  00022F85   GATEWAY   0001:00012F85 
 026D7D38  000228AE   GATEWAY   0001:000128AE 
 026D7D68  00023267   GATEWAY   0001:00013267 
 026D7D88  0002335A   GATEWAY   0001:0001335A 
 026D7D98  00010000   GATEWAY   0001:00000000 
 026D7DB8  000104B9   GATEWAY   0001:000004B9 
 026D7DC4  000103E9   GATEWAY   0001:000003E9 
 026D7DEC  00020000   GATEWAY   0001:00010000 
 026D7E08  00010F73   GATEWAY   0001:00000F73 
 026D7E28  1E630BD4   LIBC066   0001:00010BD4 
 026D7E48  000111C0   GATEWAY   0001:000011C0 
 026D7E98  00028305   GATEWAY   0001:00018305 
 026D7F08  00011353   GATEWAY   0001:00001353 
 026D7F48  00024130   GATEWAY   0001:00014130 
 026D7F88  1EA036B1   FXUTIL    0001:000036B1 
 026D7F98  1E650438   LIBC066   0001:00030438 
 026D7FA8  000180F8   GATEWAY   0001:000080F8 
 026D7FB8  1E65A5A3   LIBC066   0001:0003A5A3 
 026D7FD8  1E666D10   LIBC066   0001:00046D10 
 026D7FF8  1FFECE38   DOSCALL1  0004:0000CE38  DOS32R3EXITADDR

______________________________________________________________________

 Stack Contents from ESP-10C to Stack Base  (ESP = 026D7C2C)
______________________________________________________________________

 --addr--   --------------dwords---------------   -----chars------
 026D7B20 : 00000000 00D06ECB 00000000 00000000 : .....n..........
 026D7B30 : 000006FC 00000000 00000000 00000000 : ................
 026D7B40 : 00000000 000037E2 00000000 00000000 : .....7..........
 026D7B50 : 00000000 00000000 00000000 00000000 : ................
 026D7B60 : 00000000 00000000 00000000 00000000 : ................
 026D7B70 : 0000150B 00000053 00000053 00000002 : ....S...S.......
 026D7B80 : 00000001 18E30000 026D7E74 00000000 : ........t~m.....
 026D7B90 : 00000002 006109D9 1E54573C 0000005B : ......a.<WT.[...
 026D7BA0 : 00010246 026D7C2C 00000053 00010069 : F...,|m.S...i...
 026D7BB0 : 1E511BDC 00000000 00001018 006137BA : ..Q..........7a.
 026D7BC0 : 1E51B6A7 026D7C04 1E51534A 00010069 : ..Q..|m.JSQ.i...
 026D7BD0 : 1E514F82 00000000 1E557913 FFFFFFFF : .OQ......yU.....
 026D7BE0 : FFFFFFFF 00000000 1E55792C FFFFFFFF : ........,yU.....
 026D7BF0 : FFFFFFFF 000050D9 FFFFFFFF 00010069 : .....P......i...
 026D7C00 : 00000000 FFFFFFFF 1E557CEF FFFFFFFF : .........|U.....
 026D7C10 : FFFFFFFF 00000000 00000000 00000000 : ................
 026D7C20 : 00000034 026D7E74 1E54572F 1E545527 : 4...t~m./WT.'UT.
 026D7C30 : 02000008 026D7C00 00000000 00000000 : .....|m.........
 026D7C40 : 00000000 00000000 00000000 00000000 : ................
 026D7C50 : 00000000 00000000 026D7E74 006137AE : ........t~m..7a.
 026D7C60 : 020AA8C0 006109D9 1E555A66 026D7C8C : ......a.fZU..|m.
 026D7C70 : 026D7C9C 00000002 026D7CAC 006137AE : .|m......|m..7a.
 026D7C80 : 00000000 00000000 00000000 00000200 : ................
 026D7C90 : B1B2F748 00000001 026D7EB4 026D028C : H........~m...m.
 026D7CA0 : 020AA8C0 026D7C70 026D7E74 006102AE : ....p|m.t~m...a.
 026D7CB0 : 00000000 006109D9 026D7E74 026D7E74 : ......a.t~m.t~m.
 026D7CC0 : 026D7D14 020AA8C0 00022B11 B1B2F748 : .}m......+..H...
 026D7CD0 : 020AA8C0 00000002 006137AE 02000008 : .........7a.....
 026D7CE0 : 020AA8C0 00000034 026D7E74 006137AE : ....4...t~m..7a.
 026D7CF0 : 026D7E74 006137AE 00022E4D 006137AE : t~m..7a.M....7a.
 026D7D00 : 026D7D18 026D7E74 00000020 1EA02561 : .}m.t~m. ...a%..
 026D7D10 : 006137C2 00000001 00000000 00000089 : .7a.............
 026D7D20 : 006137AE 026D7E74 00022F85 006137AE : .7a.t~m../...7a.
 026D7D30 : 026D7E74 006137AE 000228AE 006137AE : t~m..7a..(...7a.
 026D7D40 : 006137C2 026D7E8C 00000001 04000008 : .7a..~m.........
 026D7D50 : 48F7B2B1 00207D06 026D7E74 006137AE : ...H.} .t~m..7a.
 026D7D60 : 026D7ECA 006137AE 00023267 006137AE : .~m..7a.g2...7a.
 026D7D70 : 026D7E74 026D7E74 026D7E74 006137AE : t~m.t~m.t~m..7a.
 026D7D80 : 026D7E74 006137AE 0002335A 006137AE : t~m..7a.Z3...7a.
 026D7D90 : 026D7E74 00000005 00010000 00000000 : t~m.............
 026D7DA0 : 00000000 026D7E74 026D7E74 006137AE : ....t~m.t~m..7a.
 026D7DB0 : 026D7E74 006137AE 000104B9 006137AE : t~m..7a......7a.
 026D7DC0 : 026D7E74 000103E9 0000000F 00000005 : t~m.............
 026D7DD0 : 026D7DF8 026D7DF4 00000000 00000000 : .}m..}m.........
 026D7DE0 : 00000000 00000000 00000042 00020000 : ........B.......
 026D7DF0 : 00000001 026D7E24 026D7E74 0000000F : ....$~m.t~m.....
 026D7E00 : 026D7E34 00000042 00010F73 026D7E34 : 4~m.B...s...4~m.
 026D7E10 : 006137A0 026D7E74 00000000 00000000 : .7a.t~m.........
 026D7E20 : 01ED0000 026D7E44 1E630BD4 1890DBE0 : ....D~m...c.....
 026D7E30 : 00000008 00000042 00000000 026D7E74 : ....B.......t~m.
 026D7E40 : 00000001 026D7EEC 000111C0 006137A0 : .....~m......7a.
 026D7E50 : 00000042 026D7E74 004F0178 00000000 : B...t~m.x.O.....
 026D7E60 : 00000000 0012FE80 00613798 00000000 : .........7a.....
 026D7E70 : 004F0000 00000000 00000000 00000000 : ..O.............
 026D7E80 : 00000000 00000000 020AA8C0 00000000 : ................
 026D7E90 : 00000001 00000000 00028305 00000001 : ................
 026D7EA0 : 00000008 000004B0 00000000 00000000 : ................
 026D7EB0 : 00000034 00000000 00000005 1D080E00 : 4...............
 026D7EC0 : 02005449 147789B3 00000008 00000000 : IT....w.........
 026D7ED0 : 00000000 00000000 00000000 00000000 : ................
 026D7EE0 : 00000000 00000000 00000000 004F013C : ............<.O.
 026D7EF0 : 004F0000 026D7F24 00000042 006137A0 : ..O.$.m.B....7a.
 026D7F00 : 026D7F68 0000000F 00011353 006137A0 : h.m.....S....7a.
 026D7F10 : 00000042 00000005 00000000 00000000 : B...............
 026D7F20 : 00000000 00000001 006137A0 000005EA : .........7a.....
 026D7F30 : 026D7F68 026D7F40 00000000 00000005 : h.m.@.m.........
 026D7F40 : 006137A0 00000005 00024130 00000042 : .7a.....0A..B...
 026D7F50 : 006137A0 00000005 00000002 00000000 : .7a.............
 026D7F60 : 00000000 00000000 00000042 00000000 : ........B.......
 026D7F70 : 00000000 00000000 0057D858 026D7FA4 : ........X.W...m.
 026D7F80 : 00000000 026D7FB4 1EA036B1 00000005 : ......m..6......
 026D7F90 : 0000001D 026D7FB4 1E650438 026D7FD4 : ......m.8.e...m.
 026D7FA0 : 00000000 026D7FD4 000180F8 20040CC0 : ......m........ 
 026D7FB0 : 026D7FD4 026D7FF4 1E65A5A3 0057D858 : ..m...m...e.X.W.
 026D7FC0 : 00000000 00000000 00000000 00000000 : ................
 026D7FD0 : 00000000 FFFFFFFF 1E666D10 00000000 : .........mf.....
 026D7FE0 : 00000000 00000000 00000000 00000000 : ................

______________________________________________________________________

 Memory addressed by EAX (18E30000) for 256 bytes
______________________________________________________________________

 --addr--   -----dwords------   ---------bytes---------   -chars--
 18E30000 : 00000000 00000000 : 00 00 00 00 00 00 00 00 : ........
 18E30008 : 02980000 00000001 : 00 00 98 02 01 00 00 00 : ........
 18E30010 : 02980000 02980000 : 00 00 98 02 00 00 98 02 : ........
 18E30018 : 00000000 00000000 : 00 00 00 00 00 00 00 00 : ........
 18E30028 : 2 lines not printed duplicate the line above
 18E30030 : 40430C01 1E0D87A8 : 01 0C 43 40 A8 87 0D 1E : ..C@....
 18E30038 : 00200009 0F070101 : 09 00 20 00 01 01 07 0F : .. .....
 18E30040 : 1E0D87D0 00000011 : D0 87 0D 1E 11 00 00 00 : ........
 18E30048 : 11090101 1E0D87D0 : 01 01 09 11 D0 87 0D 1E : ........
 18E30050 : 00000011 070F0101 : 11 00 00 00 01 01 0F 07 : ........
 18E30058 : 1E0D87D0 00000011 : D0 87 0D 1E 11 00 00 00 : ........
 18E30060 : 09110101 1E0D87D0 : 01 01 11 09 D0 87 0D 1E : ........
 18E30068 : 00000011 22010101 : 11 00 00 00 01 01 01 22 : ......."
 18E30070 : 1E0D87D0 00000000 : D0 87 0D 1E 00 00 00 00 : ........
 18E30078 : 24030101 1E0D87D0 : 01 01 03 24 D0 87 0D 1E : ...$....
 18E30080 : 00000000 21000101 : 00 00 00 00 01 01 00 21 : .......!
 18E30088 : 1E0D87D8 00800400 : D8 87 0D 1E 00 04 80 00 : ........
 18E30090 : 21000101 1E0D87E0 : 01 01 00 21 E0 87 0D 1E : ...!....
 18E30098 : 00800400 0F070101 : 00 04 80 00 01 01 07 0F : ........
 18E300A0 : 1E0D87E8 00000011 : E8 87 0D 1E 11 00 00 00 : ........
 18E300A8 : 11090101 1E0D87E8 : 01 01 09 11 E8 87 0D 1E : ........
 18E300B0 : 00000011 070F0101 : 11 00 00 00 01 01 0F 07 : ........
 18E300B8 : 1E0D87E8 00000011 : E8 87 0D 1E 11 00 00 00 : ........
 18E300C0 : 09110101 1E0D87E8 : 01 01 11 09 E8 87 0D 1E : ........
 18E300C8 : 00000011 22010101 : 11 00 00 00 01 01 01 22 : ......."
 18E300D0 : 1E0D87E8 00000000 : E8 87 0D 1E 00 00 00 00 : ........
 18E300D8 : 24030101 1E0D87E8 : 01 01 03 24 E8 87 0D 1E : ...$....
 18E300E0 : 00000000 21000101 : 00 00 00 00 01 01 00 21 : .......!
 18E300E8 : 1E0D87D8 00800080 : D8 87 0D 1E 80 00 80 00 : ........
 18E300F0 : 11000101 1E0D87EC : 01 01 00 11 EC 87 0D 1E : ........
 18E300F8 : 00000001 10000101 : 01 00 00 00 01 01 00 10 : ........

______________________________________________________________________

 Memory addressed by EBX (026D7E74) for 256 bytes
______________________________________________________________________

 --addr--   -----dwords------   ---------bytes---------   -chars--
 026D7E74 : 00000000 00000000 : 00 00 00 00 00 00 00 00 : ........
 026D7E7C : 00000000 00000000 : 00 00 00 00 00 00 00 00 : ........
 026D7E84 : 00000000 020AA8C0 : 00 00 00 00 C0 A8 0A 02 : ........
 026D7E8C : 00000000 00000001 : 00 00 00 00 01 00 00 00 : ........
 026D7E94 : 00000000 00028305 : 00 00 00 00 05 83 02 00 : ........
 026D7E9C : 00000001 00000008 : 01 00 00 00 08 00 00 00 : ........
 026D7EA4 : 000004B0 00000000 : B0 04 00 00 00 00 00 00 : ........
 026D7EAC : 00000000 00000034 : 00 00 00 00 34 00 00 00 : ....4...
 026D7EB4 : 00000000 00000005 : 00 00 00 00 05 00 00 00 : ........
 026D7EBC : 1D080E00 02005449 : 00 0E 08 1D 49 54 00 02 : ....IT..
 026D7EC4 : 147789B3 00000008 : B3 89 77 14 08 00 00 00 : ..w.....
 026D7ECC : 00000000 00000000 : 00 00 00 00 00 00 00 00 : ........
 026D7EE4 : 3 lines not printed duplicate the line above
 026D7EEC : 004F013C 004F0000 : 3C 01 4F 00 00 00 4F 00 : <.O...O.
 026D7EF4 : 026D7F24 00000042 : 24 7F 6D 02 42 00 00 00 : $.m.B...
 026D7EFC : 006137A0 026D7F68 : A0 37 61 00 68 7F 6D 02 : .7a.h.m.
 026D7F04 : 0000000F 00011353 : 0F 00 00 00 53 13 01 00 : ....S...
 026D7F0C : 006137A0 00000042 : A0 37 61 00 42 00 00 00 : .7a.B...
 026D7F14 : 00000005 00000000 : 05 00 00 00 00 00 00 00 : ........
 026D7F1C : 00000000 00000000 : 00 00 00 00 00 00 00 00 : ........
 026D7F24 : 00000001 006137A0 : 01 00 00 00 A0 37 61 00 : .....7a.
 026D7F2C : 000005EA 026D7F68 : EA 05 00 00 68 7F 6D 02 : ....h.m.
 026D7F34 : 026D7F40 00000000 : 40 7F 6D 02 00 00 00 00 : @.m.....
 026D7F3C : 00000005 006137A0 : 05 00 00 00 A0 37 61 00 : .....7a.
 026D7F44 : 00000005 00024130 : 05 00 00 00 30 41 02 00 : ....0A..
 026D7F4C : 00000042 006137A0 : 42 00 00 00 A0 37 61 00 : B....7a.
 026D7F54 : 00000005 00000002 : 05 00 00 00 02 00 00 00 : ........
 026D7F5C : 00000000 00000000 : 00 00 00 00 00 00 00 00 : ........
 026D7F64 : 00000000 00000042 : 00 00 00 00 42 00 00 00 : ....B...

______________________________________________________________________

 DLLs accessible from this process
______________________________________________________________________

 PMCTLS      C:\OS2\DLL\PMCTLS.DLL
 PMMERGE     C:\OS2\DLL\PMMERGE.DLL
 PMGPI       C:\OS2\DLL\PMGPI.DLL
 SEAMLESS    C:\OS2\DLL\SEAMLESS.DLL
 DISTORM     C:\ECS\DLL\DISTORM.DLL
 EXCEPTQ     C:\ECS\DLL\EXCEPTQ.DLL
 IPFORMAT    C:\UTIL\FIREWALL\IPFORMAT.DLL
 FIREWALL    C:\UTIL\FIREWALL\FIREWALL.DLL
 FXAUTH      C:\UTIL\FIREWALL\FXAUTH.DLL
 IPSEC       C:\UTIL\FIREWALL\IPSEC.DLL
 KLIBCCFG    C:\OS2\DLL\KLIBCCFG.DLL
 LIBC066     C:\UTIL\LIB\LIBC066.DLL
 GCC1        C:\UTIL\LIB\GCC1.DLL
 GENERATE    C:\UTIL\FIREWALL\GENERATE.DLL
 FXMASQ      C:\UTIL\FIREWALL\FXMASQ.DLL
 IPFRAG      C:\UTIL\FIREWALL\IPFRAG.DLL
 FXUTIL      C:\UTIL\FIREWALL\FXUTIL.DLL
 TCPIP32     C:\MPTN\DLL\TCPIP32.DLL
 VBE2GRAD    C:\OS2\DLL\VBE2GRAD.DLL
 PANOGREX    C:\OS2\DLL\PANOGREX.DLL
 VMAN        C:\OS2\DLL\VMAN.DLL
 GRE2VMAN    C:\OS2\DLL\GRE2VMAN.DLL
 TRUETYPE    C:\OS2\DLL\TRUETYPE.DLL
 PMUNIF      C:\OS2\DLL\PMUNIF.DLL
 PMATM       C:\OS2\DLL\PMATM.DLL
 DISPLAY     C:\OS2\DLL\DISPLAY.DLL
 LOCALE      C:\OS2\DLL\LOCALE.DLL
 SOMIR       C:\OS2\DLL\SOMIR.DLL
 SOMTC       C:\OS2\DLL\SOMTC.DLL
 IBMHMGR     C:\OS2\DLL\IBMHMGR.DLL
 BDCALL32    C:\OS2\DLL\BDCALL32.DLL
 PMBIDI      C:\OS2\DLL\PMBIDI.DLL
 HELPMGR     C:\OS2\DLL\HELPMGR.DLL
 PMEX        C:\OS2\DLL\PMEX.DLL
 BDWPCLS     C:\OS2\DLL\BDWPCLS.DLL
 SOMU        C:\OS2\DLL\SOMU.DLL
 SOM         C:\OS2\DLL\SOM.DLL
 LIBUNI      C:\OS2\DLL\LIBUNI.DLL
 IMP         C:\OS2\DLL\IMP.DLL
 SPL1B       C:\OS2\DLL\SPL1B.DLL
 PMSPL       C:\OS2\DLL\PMSPL.DLL
 PMSHLTKT    C:\OS2\DLL\PMSHLTKT.DLL
 UCONV       C:\OS2\DLL\UCONV.DLL
 SOFTDRAW    C:\OS2\DLL\SOFTDRAW.DLL
 PMVIOP      C:\OS2\DLL\PMVIOP.DLL
 PMWP        C:\OS2\DLL\PMWP.DLL
 FFST        C:\OS2\DLL\FFST.DLL
 DOSCALL1    C:\OS2\DLL\DOSCALL1.DLL

______________________________________________________________________

 End of Exception Report - report took 209 ms to generate
______________________________________________________________________

**Comment by SilvanScherrer** [comment]
same as #54 ?

**Comment by ecomstation** [comment]
Silvan i don't know if it's the same like ticket 54, i guess coders of injoy could answer that

**Comment by SilvanScherrer** [comment]
is this still happening? If so please add the symbol (dbg) files. As w/o them we can't see where it exactly crashes

**Comment by SilvanScherrer** [comment]
no followup, so I guess it's not an issue anymore. If yes please open a new ticket with all requested infos
